packages/khalti/test/khalti_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:khalti/khalti.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:test/test.dart';
+
+class _MockKhaltiService extends Mock implements KhaltiService {}
+
+void main() {
+  const _testPublicKey = '__test_public_key__';
+  final _config = KhaltiConfig(
+    platform: 'Android',
+    osVersion: '12',
+    deviceModel: 'Pixel 6 Pro',
+    deviceManufacturer: 'Google',
+    packageName: 'com.khalti.test',
+    packageVersion: '3.0.0',
+  );
+
+  group(
+    'KhaltiConfig |',
```

Comment:
Please rename this to something else, `KhaltiConfig` is not even there in the package, and these tests doesn't seem like it falls under `KhaltiConfig` test.